docs/operator-guide/install-argocd.md

```diff
@@ -87,106 +87,107 @@ Follow the steps below to install Argo CD using Helm:
+        # -- Default domain used by all components
+        ## Used for ingresses, certificates, SSO, notifications, etc.
+        domain: argocd.example.com
+      configs:
+        cm:
+          # required when SSO is enabled
+          url: "https://argocd.example.com"
+        oidc.config: |
+          issuer: https://keycloak.example/auth/realms/shared
       server:
-        replicas: 2
-        extraArgs:
-          - "--insecure"
-        env:
-          - name: ARGOCD_API_SERVER_REPLICAS
-            value: '2'
         ingress:
           enabled: true
           hosts:
-            - "argocd.<Values.global.dnsWildCard>"
-
-        rbacConfig:
-          # users may be still be able to login,
-          # but will see no apps, projects, etc...
-          policy.default: ''
-          scopes: '[groups]'
-          policy.csv: |
-            # default global admins
-            g, ArgoCDAdmins, role:admin
-
-      repoServer:
-        replicas: 2
-
-      # Deploy without sso
-      dex:
-        enabled: false
-
-      # Disabled for multitenancy env with single instance deployment
-      applicationSet:
-        enabled: false
-      ```
-
-    </details>
+            - "argocd.example.com"
+        oidc:
+          enabled: true
+        eso:
+          enabled: true
+    ```
 
     <details>
-    <summary><b>View: openshift-values.yaml</b></summary>
+    <summary><b>View: openshift values.yaml</b></summary>
 
       ```yaml
-      redis-ha:
-        enabled: true
-
-      controller:
-        enableStatefulSet: true
-
+    argo-cd:
+      global:
+        # -- Default domain used by all components
+        ## Used for ingresses, certificates, SSO, notifications, etc.
+        domain: argocd.example.com
+      configs:
+        cm:
+          # required when SSO is enabled
+          url: "https://argocd.example.com"
+        oidc.config: |
+          issuer: https://keycloak.example/auth/realms/shared
       server:
-        replicas: 2
-        extraArgs:
-          - "--insecure"
-        env:
-          - name: ARGOCD_API_SERVER_REPLICAS
-            value: '2'
         route:
           enabled: true
-          hostname: "argocd.<.Values.global.dnsWildCard>"
+          hostname: "argocd.example.com"
           termination_type: edge
           termination_policy: Redirect
-
-        rbacConfig:
-          # users may be still be able to login,
-          # but will see no apps, projects, etc...
-          policy.default: ''
-          scopes: '[groups]'
-          policy.csv: |
-            # default global admins
-            g, ArgoCDAdmins, role:admin
-
-      repoServer:
-        replicas: 2
-
-      # Deploy without sso
-      dex:
-        enabled: false
-
-      # Disabled for multitenancy env with single instance deployment
-      applicationSet:
-        enabled: false
+        oidc:
+          enabled: true
+        eso:
+          enabled: true
       ```
 
     </details>
 
-2. Run the installation:
+## Next Step
```

Comment:
maybe we need some result of this documentation? So now we have:
1) Centralized Argocd instance deployed by helm
2) add-ons repo enable and configured with argocd
3) Init componentnt (eso, kk,,) deployed